### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by BehnazDer**

### CODE REVIEW

File: src/components/content/ArticleCard.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { View, StyleSheet, Text, Image, Dimensions, TouchableOpacity } from 'react-native'
+
+const { width, height } = Dimensions.get('window')
+
+const ArticleCard = ({ item, navigation }) => {
+    console.log(item)
```
Comment (irina-baeva) [inline src/components/content/ArticleCard.js:7]: console log

File: src/screens/ArticlePage/index.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import appApiClient from '../../api/appApiClient';
+const { width, height } = Dimensions.get('window')
+
+
+
+export default function ArticlePage({route , navigation}) {
+  console.log(route.params.id)
+  const [uniqueArticle, setUniqueArticle] = useState({});
+useEffect(() => {
+  findOneArticle()
```
Comment (irina-baeva) [inline src/screens/ArticlePage/index.js:12]: do u use prettier ? and did u check yarn lint for errors in your files (do not pay attention other errors which  we are aware of)

File: src/screens/ArticlePage/index.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+}, []);
+  const findOneArticle = async () => {
+    try {
+      const response = await appApiClient.get(`/articles/${route.params.id}`);
+      setUniqueArticle(response.data);
+      console.log(response.data);
```
Comment (irina-baeva) [inline src/screens/ArticlePage/index.js:18]: console log

File: src/screens/ArticlesList/index.js [hunk 4]
```diff
@@ -1,11 +1,34 @@
-import React from 'react';
-import {View, Text} from 'react-native';
+import React, { useState, useEffect } from 'react'
+import { View, FlatList } from 'react-native'
+import ArticleCard from 'components/content/ArticleCard'
+import appApiClient from '../../api/appApiClient';
 
+export default function Content({ navigation}) {
+const [blog, setBlog] = useState([]);
```
Comment (irina-baeva) [inline src/screens/ArticlesList/index.js:7]: why blog name? if u get articles

Comment (BehnazDer) [reply src/screens/ArticlesList/index.js:7]: I had used article and articles so many times, just decided to go for sth for less confusion in my files

File: src/screens/ArticlePage/index.js (hunk 2 above)
Comment (BehnazDer) [reply src/screens/ArticlePage/index.js:12]: Yes, I ran yarn lint and there were no warnings in my files

File: src/components/content/ArticleCard.js (hunk 1 above)
Comment (BehnazDer) [reply src/components/content/ArticleCard.js]: removed

File: src/screens/ArticlePage/index.js (hunk 3 above)
Comment (BehnazDer) [reply src/screens/ArticlePage/index.js]: removed

**Pull request merged**